ospfd/ospf_vty.c

```diff
@@ -10011,19 +10011,21 @@ DEFPY(ospf_gr_helper_planned_only,
 }
 
 /* External Route Aggregation */
-DEFUN (ospf_external_route_aggregation,
-       ospf_external_route_aggregation_cmd,
-       "summary-address A.B.C.D/M [tag (1-4294967295)]",
-       "External summary address\n"
-       "Summary address prefix (a.b.c.d/m) \n"
-       "Router tag \n"
-       "Router tag value\n")
+DEFUN(ospf_external_route_aggregation, ospf_external_route_aggregation_cmd,
+      "summary-address A.B.C.D/M [tag (1-4294967295)] [metric (0-16777215)]",
```

Comment:
Ideally we should use curly braces in the command definition to allow both options to be configured in any given order.

Reply:
Addressed